pandas_bokeh/geoplot.py

```diff
@@ -143,11 +143,13 @@ def extract(row, geometry):
     for i, row in gdf.iterrows():
         geometry = row[geometry_column_name]
         if geometry.type == "Polygon":
-            gdf_new = gdf_new.append(extract(row, geometry), ignore_index=True)
+            gdf_new.append(extract(row, geometry))
 
         if geometry.type == "MultiPolygon":
             for polygon in geometry:
-                gdf_new = gdf_new.append(extract(row, polygon), ignore_index=True)
+                gdf_new.append(extract(row, polygon))
+
+    gdf_new = gpd.GeoDataFrame(gdf_new)
```

Comment:
This actually also does not need to be a GeoDataFrame, as we are dropping the geometry column just after. Making this a `pd.DataFrame` will avoid another validation of the geometries and a further (although minor) speed-up